code/model/DNProject.php

```diff
@@ -621,6 +621,78 @@ public function getRepositoryURL() {
 	}
 
 	/**
+	 * Whitelist configuration that describes how to convert a repository URL into a link
+	 * to a web user interface for that URL
+	 *
+	 * Consists of a hash of "full.lower.case.domain" => {configuration} key/value pairs
+	 *
+	 * {configuration} can either be boolean true to auto-detect both the host and the
+	 * name of the UI provider, or a nested array that overrides either one or both
+	 * of the auto-detected valyes
+	 *
+	 * @var array
+	 */
+	static private $repository_interfaces = array(
+		'github.com' => true,
+		'bitbucket.org' => true,
+		'repo.or.cz' => array(
```

Comment:
repo.or.cz is the third most common free git hosting service. Happy to take it out, but ideally this'd support all the common public hosts by default.